prometheus_api_client/prometheus_connect.py

```diff
@@ -569,6 +568,8 @@ def get_metric_aggregation(
                 'max': 6.009373
              }
         """
+        import numpy
```

Comment:
Can we put some try/except here? If a user has installed the minimal version and ties calling this function, they will get a generic import error. IMO we should explicitly mention that they’ll need to install the `[numpy]` or `[full]` version.

I _believe_ this suggestion applies for Metric, MetricPlot, and other classes as well